distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Statement_Setter.enso

```diff
@@ -40,7 +41,7 @@ fill_hole_default stmt i type_hint value = case value of
             big_decimal = NumericConverter.bigIntegerAsBigDecimal value
             stmt.setBigDecimal i big_decimal
         False -> stmt.setLong i value
-    _ : Decimal   -> stmt.setBigDecimal i value.big_decimal
+    _ : Decimal   -> stmt.setBigDecimal i (get_big_decimal value)
```

Comment:
Using `## PRIVATE` from another library seems like cheating.

Reply:
It does a little. However, this was the only workaround I could think of. But to paraphrase @radeusgd , "`## PRIVATE` is not a true `private`, it only hides the symbol from the component browser."

Reply:
Having a cheating method like `get_big_decimal` is certainly better than exposing internal constructors to every Enso user.